internal/pkg/db/redis/queries.go

```diff
@@ -242,7 +242,9 @@ func deleteObject(remover models.Remover, id string, conn redis.Conn) {
 	for _, cmd := range remover.Remove() {
 		switch cmd.Command {
 		case "ZREM":
+			_ = conn.Send(cmd.Command, cmd.Hash, cmd.Key)
 		case "SREM":
+			_ = conn.Send(cmd.Command, cmd.Hash, cmd.Key)
 		case "HDEL":
 			_ = conn.Send(cmd.Command, cmd.Hash, cmd.Key)
```

Comment:
Why duplicate these lines?  Why not `fallthrough`?

Reply:
The original code is  only available for  redis command "HDEL".  
Command "SREM" and "SREM" is not available and will not send to redis.

Reply:
That's a good idea.   
I will fix it.